src/java.base/share/classes/jdk/internal/foreign/abi/aarch64/CallArranger.java

```diff
@@ -186,15 +186,19 @@ public Bindings getBindings(MethodType mt, FunctionDescriptor cDesc, boolean for
 
     /* Replace DowncallLinker in OpenJDK with the implementation of DowncallLinker specific to OpenJ9 */
     public MethodHandle arrangeDowncall(MethodType mt, FunctionDescriptor cDesc, LinkerOptions options) {
-        // MethodHandle handle = DowncallLinker.getBoundMethodHandle(mt, cDesc, options);
-        // return handle;
-        return null;
+        if (System.getProperty("os.name").toLowerCase().contains("windows")) {
```

Comment:
Fetching the system property should not occur for every call to this method; the test should be captured in a static variable and use the same test as [`ProcessBuilder`](https://github.com/ibmruntimes/openj9-openjdk-jdk/blob/openj9/src/java.base/share/classes/java/lang/ProcessBuilder.java#L474): `System.getProperty("os.name").startsWith("Windows")`. Same comment for line 198.

Reply:
Fixed as suggested above.